src/integration/Sites.spec.ts

```diff
@@ -0,0 +1,205 @@
+        id: mockIsomerUserId,
+      })
+      await Site.create({
+        id: mockSiteId,
+        name: mockSite,
+        apiTokenName: "token",
+        jobStatus: "READY",
+        siteStatus: "LAUNCHED",
+        creatorId: mockIsomerUserId,
+      })
+      await Site.create({
+        id: "200",
+        name: mockAdminSite,
+        apiTokenName: "token",
+        jobStatus: "READY",
+        siteStatus: "LAUNCHED",
+        creatorId: mockIsomerUserId,
+      })
+      await SiteMember.create({
+        userId: mockIsomerUserId,
+        siteId: mockSiteId,
+        role: "ADMIN",
+      })
+      await Repo.create({
+        name: mockSite,
+        url: "url",
+        siteId: mockSiteId,
+      })
+    })
+    afterEach(async () => {
+      // Clean up so that different tests using
+      // the same mock user don't interfere with each other
+      await IsomerAdmin.destroy({
+        where: { userId: mockIsomerUserId },
+      })
+    })
+    it("should return list of only sites available to email user", async () => {
+      // Arrange
+      const expected = {
+        siteNames: [
+          {
+            lastUpdated: mockUpdatedAt,
+            repoName: mockSite,
+            isPrivate: mockPrivate,
+            permissions: mockPermissions,
+          },
+        ],
+      }
+      ;((genericGitHubAxiosInstance.get as unknown) as jest.Mock).mockImplementationOnce(
+        () => ({
+          data: [
+            {
+              pushed_at: mockUpdatedAt,
+              permissions: mockPermissions,
+              name: mockSite,
+              private: mockPrivate,
+            },
+            {
+              pushed_at: mockUpdatedAt,
+              permissions: mockPermissions,
+              name: mockAdminSite,
+              private: mockPrivate,
+            },
+          ],
+        })
+      )
+      ;((genericGitHubAxiosInstance.get as unknown) as jest.Mock).mockImplementationOnce(
+        () => ({
+          data: [],
+        })
+      )
+      ;((genericGitHubAxiosInstance.get as unknown) as jest.Mock).mockImplementationOnce(
+        () => ({
+          data: [],
+        })
+      )
+
+      // Act
+      const actual = await request(app).get("/")
+
+      // Assert
+      expect(actual.body).toMatchObject(expected)
+    })
+    it("should return list of all sites available for admin", async () => {
+      // Arrange
+      await IsomerAdmin.create({
+        userId: mockIsomerUserId,
+      })
+      const expected = {
+        siteNames: [
+          {
+            lastUpdated: mockUpdatedAt,
+            repoName: mockSite,
+            isPrivate: mockPrivate,
+            permissions: mockPermissions,
+          },
+          {
+            lastUpdated: mockUpdatedAt,
+            repoName: mockAdminSite,
+            isPrivate: mockPrivate,
+            permissions: mockPermissions,
+          },
+        ],
+      }
+      ;((genericGitHubAxiosInstance.get as unknown) as jest.Mock).mockImplementation(
```

Comment:
this actually impacts readability pretty significantly .-. do we have a better way around this? given that this is a test file, i'm not too concerned by it but if there's a quick way to rewrite this, i think we should